### DISCUSSION TITLE
Mouse Acceleration, Hibernate timer, Process Kill list

### DISCUSSION THREAD

**Discussion by SuperLeiyan** [hansschmucker/NVStreamer1080#3] [Ideas]
This little utility works fantastically, even perfectly restores upon waking up the pc from hibernation! I imagine it wouldn't be too hard to also restore Windows mouse acceleration state? It has always been an [annoyance ](https://github.com/moonlight-stream/moonlight-docs/wiki/Frequently-Asked-Questions#why-does-my-hosts-mouse-acceleration-change-while-streaming) that nvstream messes with your mouse acceleration setting, and it doesn't restore after wake up if you hibernated your PC from a stream session.

On the note of hibernating PC, it would be awesome if there was a configurable timer to auto hibernate after a stream ends, and possibly a kill list to auto terminate game or other background processes.

**Comment by hansschmucker** [comment]
As far as I can tell this is possible with SPI_SETMOUSESPEED, but since I haven't encountered that issue yet I'm not quite sure what you expect ... could you please describe it in more detail? Something like

- On Idle, system was configured with 2x mouse acceleration
- Stream has started, acceleration is now at 1x
- Stream has ended, mouse acceleration stays at 1x, should be 2x

or (maybe that's what you meant, I'm not quite sure)
- On Idle, system was configured with 2x mouse acceleration
- Stream has started, acceleration is now at 2x (desired is a user configurable value, in this case 1x)
- Stream has ended, acceleration is now 2x (desired)

or maybe
- On Idle, system was configured with 1x mouse acceleration
- Stream has started, acceleration is now at 1x
- Changing acceleration manually to 2x
- Stream has ended, acceleration is now 2x, should be 1x again

Just so I can unserstand what you need ^^
--hans

**Reply by SuperLeiyan** [reply]
Yeah ok, sorry I wasn't clear enough, I just meant this toggle here, I always have this disabled cause I play a lot of FPS games on my PC, but whenever nvstream starts it gets turned on, and it stays on even after a session has ended, and I need to remember to manually toggle it off every time I'm done streaming. So for my use cause, just be able to restore the state of that toggle would be enough.
![image](https://user-images.githubusercontent.com/5438286/187071117-64c83414-41f7-4324-b9ba-2cf2a8e42481.png)

**Comment by hansschmucker** [comment]
Long story short, feel free to file three seperate issues for these... can't promise when I'll get around to doing them all, but when I have some spare time I'll look into them :)
--hans
